src/dshould/basic.d

```diff
@@ -0,0 +1,328 @@
+module dshould.basic;
+
+import std.string : empty;
+import dshould.ShouldType;
+public import dshould.ShouldType : should;
+
+pure @safe unittest
```

Comment:
_[...] all functions will be exercised by unit tests using `unittest` blocks immediately **following** the function to be tested._

Reply:
Then the unittests will be spread in tiny bits all over the module, making them useless as we don't generate ddoc.

Reply:
That's not true for our GitHub repositories. Adam generates the documentation:
https://forum.dlang.org/post/qeiapxuoamaumtggpqxl@forum.dlang.org

Reply:
fixed